docs/overview/sapi.rst

```diff
@@ -131,80 +307,66 @@ You can test that your solver access is configured correctly with the
+You can see the supported regions and the solvers available in each for your
+account in your `Leap <https://cloud.dwavesys.com/leap/>`_ dashboard.
+Ocean's :ref:`sdk_index_cloud` enables you to select solvers from a particular
+region as follows:
+
+* Select a default region in your
+  :ref:`dwave-cloud-client configuration file <sdk_index_cloud>`. You can run
+  the :ref:`dwave config <cli_example_config>` CLI command with the
+  :code:`--full` option or edit an existing configuration file to set a
+  preferred region.
+* Set the appropriate environment variable (for example,
+  :code:`export DWAVE_API_REGION=eu-central-1` in a Unix shell) for your current
+  terminal or working session to select solvers from a preferred region.
 * Explicitly select the region in your code. For example, the :code:`region`
-  parameter in the code line :code:`sampler = DWaveSampler(region="na-west-1")` 
-  selects a D-Wave quantum computer located in North America. 
-
-.. note:: Keep in mind the relative priorities of configurations set at 
-   various levels (in the above bullets, the configuration file, environment 
-   variables, and explicit parameters in your code), as described in the 
-   :ref:`sdk_index_cloud` documentation. For example, while the setting of both 
-   an endpoint and region on the same level (either two lines in the configuration 
-   file or two environment variables or two explicit parameters) results in 
-   selection based on the endpoint, setting a region explicitly overrides an 
+  parameter in the code line :code:`sampler = DWaveSampler(region="na-west-1")`
+  selects a D-Wave quantum computer located in North America.
+
+.. note:: Keep in mind the relative priorities of configurations set at
+   various levels (in the above bullets, the configuration file, environment
+   variables, and explicit parameters in your code), as described in the
+   :ref:`sdk_index_cloud` documentation. For example, while the setting of both
+   an endpoint and region on the same level (either two lines in the configuration
+   file or two environment variables or two explicit parameters) results in
+   selection based on the endpoint, setting a region explicitly overrides an
    endpoint configured using an environment variable or in the configuration
-   file. 
+   file.
+
+Example: Viewing a Region's Solvers
+-----------------------------------
+
+You can use :ref:`interactive CLI <dwave_cli>` to query solvers in a particular
+region.
+
+.. code-block:: bash
+
+    $ dwave solvers --list --region eu-central-1
+    Advantage_system5.1
+
+Example: Using Configuration-File Profiles for Multiple Regions
+---------------------------------------------------------------
+
+You can set up your configuration file with profiles for multiple regions, as
+shown below.
+
+.. code-block:: bash
+
+  [defaults]
+  token = ABC-123456789123456789123456789
+
+  [europe]
+  token = DEF-987654321987654321987654321
```

Comment:
It might be confusing/misleading to re-set the token. Tokens are global, not per region.